Makefile.am

```diff
@@ -48,6 +53,7 @@ noinst_LIBRARIES = \
 	cpp/libtest.a
 
 check_PROGRAMS = \
+	cpp/util/thread_pool_test \
```

Comment:
So it's just wholly disabled for now? I'd say adding it conditionally to `TESTS` should be the next priority?